### PR TITLE
XSite Repl test environment is configured and new tests are added.

### LINKED ISSUE #531: Provide x-site cypress feature tests
### Description

Xsite tests covering:

- Cache creation wizard
- Label display on sites (data container, backups label)
- X-site cache state transfert actions

### Implementation ideas

Start a server with x-site enabled and run tests

### PR DESCRIPTION
**Pull request by andyuk1986**
Closes #531

### CODE REVIEW

**Comment by dpanshug** [comment]
In addition, it would be beneficial to incorporate the `xsite.cy.js` test for thorough testing of this PR within the GitHub Actions workflow.

**Review by dpanshug — CHANGES_REQUESTED** [review]
In addition, it would be beneficial to incorporate the `xsite.cy.js` test for thorough testing of this PR within the GitHub Actions workflow.

File: package.json [hunk 1]
```diff
@@ -20,8 +20,9 @@
     "bundle-profile:analyze": "npm run build:bundle-profile && webpack-bundle-analyzer ./stats.json",
     "clean": "rimraf dist",
     "deploy-console-to-server": "cp -r ./dist/* ./server/infinispan-server/static/console",
-    "cy:run": "cypress run --headless",
-    "cy:e2e": "cypress open"
+    "cy:run": "cypress run --headless --spec 'cypress/e2e/!(xsite.cy.js)'",
+    "cy:e2e": "cypress open",
+    "cy:run-xsite": "cypress run --headless --spec 'cypress/e2e/xsite.cy.js'"
```
Comment (dpanshug) [inline package.json:25]: The server `./run-xsite.tests.sh` stop after running, how can we then utilize this command?

**Comment by andyuk1986** [comment]
OK. I think I will change the whole test running scripts to work over docker and include also xsite tests in it. It will include run-server-for-e2e.sh as well as the pull request verification script.

**Comment by karesti** [comment]
@andyuk1986 which is the status here ?

**Comment by andyuk1986** [comment]
@karesti I am updating the PR to include the xsite tests in PR checkups too, as well as to use the latest infinispan-dev-mode.xml in one general script.

**Comment by karesti** [comment]
@andyuk1986 which is the status here 🙏🏻 ?

**Comment by andyuk1986** [comment]
Hey @karesti , at the moment this PR is on hold. I have some work in progress on my local, but there are other priorities now. As soon as I have some time, I will finish the work and will update this PR.

**Comment by karesti** [comment]
@andyuk1986 we really need these tests to assure non regression on backups

**Comment by andyuk1986** [comment]
Hey Katia, I have started to work on it. Will be ready as soon as possible.

**Comment by karesti** [comment]
@andyuk1986 thanks !!! :))

**Comment by andyuk1986** [comment]
@karesti I am closing this PR as it is really hard to resolve the conflicts after a year pause and changes from main. I have created new PR #540 which includes these changes on the clear rebased main.

**Pull request closed without merging**